### ISSUE TITLE
Parts of 2D structure disappear or have distorted graphics when bases are supposed to be highlighted

### ISSUE THREAD

**Issue by sbutkovi** [eternagame/EternaJS#634]
Testing 1.4.6 on iPhone (iOS 15.4.1). When testing the first tutorial puzzle yesterday (Intro 1), I happened across these graphics twice (for this screenshot, I had "clash-free layout" turned on):

![pic1](https://user-images.githubusercontent.com/35237235/161400315-b0492e91-b7f2-4ec1-ac46-81bb85328e34.PNG)

When I tested the first tutorial puzzle today, I happened across these graphics twice (for this screenshot, I did not have "clash-free layout" turned on):

![pic2](https://user-images.githubusercontent.com/35237235/161400319-730b57c2-044d-47b9-873b-e7c1219d1a98.png)

I've inferred that it involves when the puzzle is trying to highlight bases, as it appears to also happen in other puzzles, such as the tutorial puzzle RNA Structure 1:

![pic3](https://user-images.githubusercontent.com/35237235/161400323-b96defa4-efda-4f90-b752-be7e84ae60aa.png)

This bug appears to occur regardless of whether I am logged in or anonymous. Additionally, it does not appear to happen in the website version of the game, at least for the puzzles shown in the attached screenshots.

**Comment by sbutkovi** [comment]
Adding additional screenshots to aid in troubleshooting. These screenshots were taken simultaneously.

![Image from iOS](https://user-images.githubusercontent.com/35237235/161887042-2152bb73-c84d-49a2-a672-4f8eb1fc06e6.png)

<img width="1157" alt="Screen Shot 2022-04-05 at 9 52 15 PM" src="https://user-images.githubusercontent.com/35237235/161887057-a89b0b53-ea9d-4cc9-8a75-3d9fce99bd50.png">

**Comment by sbutkovi** [comment]
An additional screenshot from the mobile app. Here, I had the "simpler, less-animated graphics" option turned on. Note that only the final base (15) appears to be highlighted. Additionally, there appear to be additional, unexpected "layered" images - the textboxes, various graphical elements (like the help, hint, and home buttons). Some of these symptoms match with what I observed in the first screenshot I posted within this issue, but not all of them do.

![Image from iOS](https://user-images.githubusercontent.com/35237235/161888530-f33de2e5-67b2-43a2-bf05-24bd733a0b02.png)
